### ISSUE TITLE
Cache not updated after changing the value?

### ISSUE THREAD

**Issue by vinzen** [huacnlee/rails-settings-cached#87]
Hi, I updated my project from Rails 3.2.x to Rails 4.2.5. One of my unit test failed because there is something wrong with the settings cache.

```
test "should return 503" do
    puts Setting.amazon_on # it's true
    Setting.amazon_on = false
    puts Setting.amazon_on # it's true
    get :show, {:uid => 'uid', :content => 1} # my controller read true
    assert @response.status == 503
end
```

I'm using version 0.5.3 (previously 0.2.4)

**Comment by wvengen** [comment]
I'm having a similar problem when upgrading from 0.4.2 to 0.5.3.

**Comment by wvengen** [comment]
Looks like a duplicate of #73.

**Comment by jpbullalayao** [comment]
I believe this is the case for 0.6.2 as well. I use `Setting.get_all` after changing the value in the DB, and `.get_all` gives me the old value

**Comment by KevinSjoberg** [comment]
@jpbullalayao I've addressed the issue in #98.
